iaso/exports/parquet.py

```diff
@@ -0,0 +1,318 @@
+    dsn = connection.get_connection_params()
+
+    con = duckdb.connect()
+    attach_sql = f"""
+        INSTALL postgres;
+        LOAD postgres;
+        ATTACH 'dbname={dsn["dbname"]} host={dsn["host"]} user={dsn["user"]} password={dsn["password"]}' AS pg (TYPE postgres, READ_ONLY);
+
+    """
+    con.execute(attach_sql)
+
+    logger.info(f"exporting parquet : {output_file_path} \n\n {full_sql}")
+    parquet_export_sql = f"""
+        COPY (
+            SELECT * FROM postgres_query('pg', $$ {full_sql} $$)
+        ) TO '{output_file_path}' (FORMAT PARQUET)
+    """
+
+    con.execute(parquet_export_sql)
+
+    row_count = con.execute(f"SELECT COUNT(*) FROM '{output_file_path}'").fetchone()[0]
+    col_count = len(con.execute(f"DESCRIBE SELECT * FROM '{output_file_path}'").fetchall())
+    con.close()
+    duration = time.perf_counter() - start
+    size_mb = os.path.getsize(output_file_path) / (1024 * 1024)
+    logger.warn(
+        f"dumped to {output_file_path} took {duration:.3f} seconds for {row_count} records and {col_count} columns, final file size {size_mb:.2f} Mb"
+    )
+
+
+def build_submission_annotations():
+    model_fields = [
+        "id",
+        "form_id",
+        "period",
+        "uuid",
+        "org_unit_id",
+        "org_unit__name",
+        "org_unit__source_ref",
+        "entity_id",
+        "created_at",
+        "source_created_at",
+        "created_by__username",
+        "created_by_id",
+        "updated_at",
+        "source_updated_at",
+        "last_modified_by__username",
+        "last_modified_by_id",
+        "deleted",
+        "export_id",
+        "correlation_id",
+    ]
+    model_prefix = "iaso_subm_"
+
+    # TODO is ref submission
+
+    # accuracy, attributes,
+    # device, device_id, entity,
+    # form_version, form_version_id,
+    # last_export_success_at,
+    # name,
+    # orgunitchangerequest, orgunitreferenceinstance,
+    # planning, planning_id, project, project_id,
+    # storage_log_entries, to_export
+    # Status,
+    # Entité,
+
+    prefixed_fields = {f"{model_prefix}{normalize_field_name(f)}": F(f) for f in model_fields}
+
+    # less standard fields that needs some functions or more complex calculation
+    prefixed_fields[f"{model_prefix}is_reference"] = Exists(
+        m.OrgUnitReferenceInstance.objects.filter(org_unit_id=OuterRef("org_unit_id"), instance_id=OuterRef("pk"))
+    )
+    prefixed_fields[f"{model_prefix}form_version_id"] = KeyTextTransform("_version", "json")
+    prefixed_fields[f"{model_prefix}longitude"] = ST_X(F("location"))
+    prefixed_fields[f"{model_prefix}latitude"] = ST_Y(F("location"))
+    prefixed_fields[f"{model_prefix}altitude"] = ST_Z(F("location"))
+    prefixed_fields[f"{model_prefix}accuracy"] = F("accuracy")
+    return prefixed_fields
+
+
+def build_submissions_queryset(qs, form_id):
+    form = m.Form.objects.get(pk=form_id)
```

Comment:
Since you have already fetched the form in the API, you can simply pass it to avoid fetching it twice
```suggestion
def build_submissions_queryset(qs, form):
```